components/common/listingCart/ListingCartModal.vue

```diff
@@ -0,0 +1,224 @@
+              " />
+          </div>
+          <div class="pt-3 has-text-grey">-Or-</div>
+          <div class="pt-3">Fixed Price</div>
+          <ListingCartPriceInput
+            v-model="fixedPrice"
+            check
+            class="pt-2"
+            @confirm="setFixedPrice" />
+        </div>
+        <div class="py-2">
+          <ListingCartItem
+            v-for="nft in listingCartStore.itemsInChain"
+            :key="nft.id"
+            :nft="nft" />
+        </div>
+        <div
+          class="is-flex border-top is-justify-content-space-between py-4 px-6">
+          Potential Earnings
+          <div class="is-flex">
+            <span class="ml-2 has-text-grey"
+              >{{ totalNFTsPrice.toFixed(4) }} {{ chainSymbol }}</span
+            >
+            <span class="has-text-weight-bold ml-2"> ${{ priceUSD }} </span>
+          </div>
+        </div>
+
+        <div class="is-flex is-justify-content-space-between pb-5 px-6">
+          <NeoButton
+            :disabled="!!listingCartStore.incompleteListPrices"
+            :label="confirmListingLabel"
+            variant="k-accent"
+            no-shadow
+            class="is-flex is-flex-grow-1 py-5"
+            @click.native="confirm" />
+        </div>
+      </div>
+    </NeoModal>
+  </div>
+</template>
+<script setup lang="ts">
+import { Interaction } from '@kodadot1/minimark/v1'
+import ListingCartPriceInput from '@/components/common/listingCart/ListingCartPriceInput.vue'
+import { prefixToToken } from '@/components/common/shoppingCart/utils'
+import IdentityItem from '@/components/identity/IdentityItem.vue'
+import { NeoButton, NeoModal } from '@kodadot1/brick'
+import { usePreferencesStore } from '@/stores/preferences'
+import { TokenToList } from '@/composables/transaction/types'
+import { useListingCartStore } from '@/stores/listingCart'
+import { calculateBalance } from '@/utils/format/balance'
+import { warningMessage } from '@/utils/notification'
+import { useFiatStore } from '~/stores/fiat'
+import { calculateExactUsdFromToken } from '~/utils/calculation'
+const { isLogIn, accountId } = useAuth()
+const { urlPrefix } = usePrefix()
+const preferencesStore = usePreferencesStore()
+const listingCartStore = useListingCartStore()
+const { transaction, isLoading, status } = useTransaction()
+const { $i18n } = useNuxtApp()
+
+const { chainSymbol } = useChain()
+const fixedPrice = ref()
+const floorPricePercentAdjustment = ref(1)
+function setFixedPrice() {
+  listingCartStore.setFixedPrice(fixedPrice.value)
+  fixedPrice.value = undefined
+}
+const fiatStore = useFiatStore()
+const priceUSD = computed(() =>
+  calculateExactUsdFromToken(
+    totalNFTsPrice.value,
+    Number(fiatStore.getCurrentTokenValue(prefixToToken[urlPrefix.value]))
+  )
+)
+
+const totalNFTsPrice = computed(() =>
+  listingCartStore.itemsInChain.reduce((acc, nft) => {
+    return +(acc + Number(nft.listPrice || 0)).toFixed(4)
```

Comment:
explain please. What is the first `+` doing?

we have utils for `sum`, consider using them

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus

It's not sum, it is to convert it to a number since toFixed converts it to a string. and wrapping it with Number again looks odd

But basically it is the same as
`Number((acc + Number(nft.listPrice || 0)).toFixed(4))`

Reply:
@kodadot/internal-dev  please weigh in on this
I personally prefer clarity over shorter line and think  that using `Number()` is preferable

please vote
:rocket:  for `+`
:heart:  for `Number()`

Reply:
I can use any. It's better for us to update our STYLE_GUIDE. 
It is to avoid a lot of re-editting in code review. 
I noticed that without the proper documentation on this, new developers like me will save a lot of time if we had clear instructions for these. These include all the classes and minor changes.

I do appreciate cleaner code and I have nothing against this. 
I'm just saying it would be of big help and a time saver if we know what are the to dos and not to dos.

Reply:
I'm open to any approach, but I suggest we update our STYLE_GUIDE. This can minimize extensive revisions during code reviews. Having thorough documentation on this will greatly benefit newcomers like myself. It should encompass class definitions, stylistic adjustments, and other minor modifications.

While I value clean code and have no reservations about it, having clear guidelines on dos and don'ts would be immensely helpful and time-efficient.

Reply:
for this PR please change it to `Number`
 if you want to add an entry about this in the style_guide, please open an issue about that
 
 thank you